frontend/views/containers/dashboard/GroupMembersAllModal.vue

```diff
@@ -1,18 +1,227 @@
 <template lang='pug'>
-modal-base-template(:fullscreen='true')
-  | TODO: Implement Group Members modal
+modal-base-template.has-background(ref='modal' :fullscreen='true')
+  .c-container
+    .c-header
+      i18n.is-title-2.c-title(tag='h2') Group members
+
+    .card.c-card
+      form(
+        @submit.prevent='submit'
+      )
+        label.field
+          .input-combo
+            .is-icon(:aria-label='L("Search")')
+              i.icon-search
+            input.input(
+              type='text'
+              name='search'
+              data-test='search'
+              placeholder='Search...'
+              v-model='form.search'
+            )
+
+        i18n.c-member-count.has-text-1(
+          v-if='form.search'
+          tag='div'
+          :args='{ searchCount: Object.keys(searchResult).length, searchTerm: form.search}'
+        ) Showing {searchCount} result for "{searchTerm}"
+
+        i18n.c-member-count.has-text-1(
+          v-else
+          tag='div'
+          :args='{ groupMembersCount }'
+        ) {groupMembersCount} members
+
+      table.table.c-table(v-if='searchResult')
+        transition-group.c-group-list(name='slide-list' tag='tbody')
+          tr.c-group-member(
```

Comment:
This for me is a list of results, not a table. In terms on A11Y there are some issues here (ex: using `display: flex` in a table element removes its semantics (the same way we use `display: none` to hide something))

So, I would advise to just replace `table` for `ul li` and everything gets easier.